frontend/javascripts/oxalis/view/action-bar/toolbar_view.js

```diff
@@ -156,6 +170,49 @@ function OverwriteModeSwitch({ isControlPressed, isShiftPressed, visible }) {
   );
 }
 
+function AdditionalSkeletonModesButtons() {
+  const dispatch = useDispatch();
+  const isMergerModeEnabled = useSelector(
+    state => state.temporaryConfiguration.isMergerModeEnabled,
+  );
+  const isNewNodeNewTreeModeOn = useSelector(state => state.userConfiguration.newNodeNewTree);
+  const toggleNewNodeNewTreeMode = () =>
+    dispatch(updateUserSettingAction("newNodeNewTree", !isNewNodeNewTreeModeOn));
+  const toggleMergerMode = () => dispatch(setMergerModeEnabledAction(!isMergerModeEnabled));
+  const activeButtonStyle = { ...narrowButtonStyle, borderColor: "#1890ff" };
+  const newNodeNewTreeModeButtonStyle = isNewNodeNewTreeModeOn
+    ? activeButtonStyle
+    : narrowButtonStyle;
+  const mergerModeButtonStyle = isMergerModeEnabled ? activeButtonStyle : narrowButtonStyle;
+
+  return (
+    <React.Fragment>
+      <Tooltip title="Toggle the Single node Tree (soma clicking) mode">
```

Comment:
I know that has been 

```suggestion
      <Tooltip title="Toggle the Single node Tree (soma clicking) mode – If enabled, each node creation will create a new tree.">
```